.gitignore

```diff
@@ -2,3 +2,5 @@
 *.pyc
 venv/
 *.env
+*.coverage
+htmlcov
```

Comment:
We should see if there's a better way to look at this testing... on the to do list.  Currently we're at 46% test coverage, but this is testing the ENTIRE application including all built in modules.